contrib/packaging/docker/init-container.sh

```diff
@@ -1,6 +1,18 @@
 #!/bin/sh
 
-cilium cleanup -f
+# Check for CLEAN_CILIUM_BPF_STATE and CLEAN_CILIUM_STATE
+# is there for backwards compatibility as we've used those
+# two env vars in our old kubernetes yaml files.
```

Comment:
I think it would be good to come up with a timeline for getting rid of the old variable checks here.